source/modulo_core/modulo_core/translators/message_readers.py

```diff
@@ -74,6 +76,27 @@ def read_message(state: StateT, message: MsgT) -> StateT:
                     state.set_torques(message.effort)
             except Exception as e:
                 raise MessageTranslationError(f"{e}")
+        elif isinstance(message, trajectory.JointTrajectory) and isinstance(state, sr.JointTrajectory):
+            try:
+                state.set_joint_names(list(message.joint_names))
+                time_from_start = 0
+                for i, point in enumerate(message.points):
+                    dir(point.time_from_start)
```

Comment:
What is this for?

Reply:
hm, leftover from debugging, good catch, thanks